managed/services/inventory/services.go

```diff
@@ -346,3 +346,85 @@ func (ss *ServicesService) Remove(ctx context.Context, id string, force bool) er
 
 	return nil
 }
+
+// AddCustomLabels adds or replaces (if key exists) custom labels for a service.
+func (ss *ServicesService) AddCustomLabels(ctx context.Context, req *inventorypb.AddCustomLabelsRequest) (*inventorypb.AddCustomLabelsResponse, error) {
+	err := ss.db.InTransactionContext(ctx, nil, func(tx *reform.TX) error {
```

Comment:
To eliminate shadowing:
```suggestion
	errTx := ss.db.InTransactionContext(ctx, nil, func(tx *reform.TX) error {
```